modules/redis/charts.go

```diff
@@ -0,0 +1,190 @@
+package redis
+
+import "github.com/netdata/go-orchestrator/module"
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+	// Dims is an alias for module.Dims
+	Dims = module.Dims
+)
+
+const (
+	keysRedisChartId = "keys_redis"
+)
+
+var charts = Charts{
+	{
+		ID:    "operations",
+		Title: "Operations",
+		Units: "operations/s",
+		Fam:   "operations",
+		Ctx:   "redis.operations",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "total_commands_processed",
+				Name: "commands",
+				Algo: module.Incremental,
+			},
+			{ID: "instantaneous_ops_per_sec", Name: "operations"},
+		},
+	},
+	{
+		ID:    "hit_rate",
+		Title: "Hit rate",
+		Units: "percentage",
+		Fam:   "hits",
+		Ctx:   "redis.hit_rate",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "hit_rate", Name: "rate"},
+		},
+	},
+	{
+		ID:    "memory",
+		Title: "Memory utilization",
+		Units: "KiB",
+		Fam:   "memory",
+		Ctx:   "redis.memory",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "used_memory",
+				Name: "total",
+				Div:  1024,
+			},
+			{
+				ID:   "used_memory_lua",
+				Name: "lua",
+				Div:  1024,
+			},
+		},
+	},
+	{
+		ID:    "net",
+		Title: "Bandwidth",
+		Units: "kilobits/s",
+		Fam:   "network",
+		Ctx:   "redis.net",
+		Type:  module.Area,
+		Dims: Dims{
+			{ID: "total_net_input_bytes", Name: "in", Algo: module.Incremental, Mul: 8, Div: 1000},
+			{ID: "total_net_output_bytes", Name: "out", Algo: module.Incremental, Mul: -8, Div: 1000},
+		},
+	},
+	{
+		ID:    keysRedisChartId,
+		Title: "Keys per Database",
+		Units: "keys",
+		Fam:   "keys",
+		Ctx:   "redis.keys",
+		Type:  module.Line,
```

Comment:
`module.Line` is the default, please remove (true for all charts with type line charts)